h/h_api/bulk_api/entry_point.py

```diff
@@ -110,7 +110,9 @@ def _string_to_lines(string):
         :param string: A string to split
         :return: A generator of strings containing single lines
         """
-        return string.strip().split("\n")
+
+        lines = (line.strip() for line in string.strip().split("\n"))
+        return (line for line in lines if line)
```

Comment:
Turns out JSON does care about new lines... I've no idea what's going on anymore. Anywho. Best to scrub out empty lines in any case.